nemo/export/trt_llm/converter/model_converter.py

```diff
@@ -78,6 +80,21 @@ def prompt_convert(prompt_config, prompt_weights):
     return vtokens_embeddings
 
 
+def create_common_export_config(nemo_model_config, decoder_type, fp8_quantized=False, fp8_kvcache=False):
+    is_mcore = nemo_model_config.get("mcore_gpt", False)
```

Comment:
I dont think we actually support legacy non-mcore nemo models. Maybe you can set this to false and eventually this can get cleaned up.

Reply:
I suppose it will be refactored once the [mcore export](https://github.com/NVIDIA/NeMo/pull/10238) module is completed.